.devcontainer/devcontainer.json

```diff
@@ -7,7 +7,7 @@
   "containerEnv": { "DEVCONTAINER": "1" },
   // Port 5683 udp is used by Shelly integration
   "appPort": ["8123:8123", "5683:5683/udp"],
-  "runArgs": ["-e", "GIT_EDITOR=code --wait"],
+  "runArgs": ["-e", "GIT_EDITOR=code --wait", "--privileged"],
```

Comment:
If this change is needed please do it on a separate PR with reasoning why

Reply:
Thanks, that was a mistake. I undid the change.